### PR TITLE
0.8 preview

### PR DESCRIPTION
**Pull request by atotic**

### CODE REVIEW

**Comment by atotic** [comment]
I am the author.

File: bower.json [hunk 1]
```diff
@@ -26,8 +26,12 @@
     "google"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#>=0.5.1 <1.0.0",
-    "core-ajax": "Polymer/core-ajax#>=0.5.1 <1.0.0",
-    "google-apis": "GoogleWebComponents/google-apis#~0.4.2"
+    "polymer": "Polymer/polymer#0-8.preview",
```
Comment (ebidel) [inline bower.json]: 0.8-preview

File: bower.json [hunk 2]
```diff
@@ -26,8 +26,12 @@
...
+    "google-apis": "GoogleWebComponents/google-apis#0.8-preview",
+    "iron-ajax": "PolymerElements/iron-ajax#master"
```
Comment (ebidel) [inline bower.json]: let's get a version on this.

File: demo.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <title>google-chart Demo</title>
 
-  <script src="../webcomponentsjs/webcomponents.min.js"></script>
+  <script src="../webcomponentsjs/webcomponents.js"></script>
```
Comment (ebidel) [inline demo.html]: lite?

File: demo.html [hunk 4]
```diff
@@ -90,6 +104,38 @@
     }, 3000);
   </script>
 
+  <p>Here's a pie chart with an area selection:</p>
+
+  <div id="selection-demo">
+    <google-chart
+      type='pie'
```
Comment (ebidel) [inline demo.html]: type="pie"

File: demo.html [hunk 5]
```diff
@@ -90,6 +104,38 @@
...
+      cols='[{"label": "Month", "type": "string"},{"label": "Days", "type": "number"}]'
+      rows='[["Jan", 31],["Feb", 28],["Mar", 31],["Apr", 30],["May", 31],["Jun", 30]]'>
+    </google-chart>
+    <div id="selection-display">
+      Selected row: <span id="selection-label">None</span>.
+    </div>
+  </div>
+
+  <script>
+    document.addEventListener('polymer-ready', function() {
```
Comment (ebidel) [inline demo.html]: `polymer-ready` is gone in 0.8, right?

File: demo.html [hunk 6]
```diff
@@ -253,5 +299,59 @@
     rows='[["Jan", 31],["Feb", 28],["Mar", 31],["Apr", 30],["May", 31],["Jun", 30]]'>
   </google-chart>
 
+  <p>Here's a table chart:</p>
+
+  <google-chart
+    type='table'
```
Comment (ebidel) [inline demo.html]: type="table"

File: demo.html [hunk 7]
```diff
@@ -253,5 +299,59 @@
...
+           ["2004", 1000, 400],
+           ["2005", 1170, 460],
+           ["2006", 660, 1120],
+           ["2007", 1030, 540]]'>
+  </google-chart>
+
+  <p>Here are three gauges:</p>
+
+  <google-chart
+      type='gauge'
```
Comment (ebidel) [inline demo.html]: type="gauge"

File: demo.html [hunk 8]
```diff
@@ -253,5 +299,59 @@
...
+                  "yellowFrom":75,
+                  "yellowTo": 90,
+                  "minorTicks": 5}'>
+  </google-chart>
+
+  <p>Here are three gauges with random data that change every three seconds:</p>
+
+  <google-chart
+      id="mutating_gauge"
+      type='gauge'
```
Comment (ebidel) [inline demo.html]: type="gauge"

File: demo.html [hunk 9]
```diff
@@ -253,5 +299,59 @@
...
+           ["2006", 660, 1120],
+           ["2007", 1030, 540]]'>
+  </google-chart>
+
+  <p>Here are three gauges:</p>
+
+  <google-chart
+      type='gauge'
+      data='[["Label", "Value"],["Memory", 80],["CPU", 55],["Network", 68]]'
+      options = '{"width": 400,
```
Comment (ebidel) [inline demo.html]: `options='{`

File: demo.html [hunk 10]
```diff
@@ -253,5 +299,59 @@
...
+                  "height": 120,
+                  "redFrom": 90,
+                  "redTo": 100,
+                  "yellowFrom":75,
```
Comment (ebidel) [inline demo.html]: "yellowFrom": 75,

File: google-chart.html [hunk 11]
```diff
@@ -41,269 +41,357 @@
...
+       * };</pre>
+       * See <a href="https://google-developers.appspot.com/chart/interactive/docs/gallery">Google Visualization API reference (Chart Gallery)</a>
+       * for the options available to each chart type.
+       *
+       * @attribute options
+       * @type object
+       */
+      options: {
+        type: Object,
+        value: function() { return {}}
```
Comment (ebidel) [inline google-chart.html]: `return {}; }`

File: google-chart.html [hunk 12]
```diff
@@ -41,269 +41,357 @@
...
+       *  {label: "Value", type: "number"}]</pre>
+       * See <a href="https://google-developers.appspot.com/chart/interactive/docs/reference#DataTable_addColumn">Google Visualization API reference (addColumn)</a>
+       * for column definition format.
+       *
+       * @attribute cols
+       * @type array
+       */
+      cols: {
+        type: Array,
+        value: function() { return []}
```
Comment (ebidel) [inline google-chart.html]: `return []; }`

File: google-chart.html [hunk 13]
```diff
@@ -41,269 +41,357 @@
...
+       *  ["Category 2", 1.1]]</pre>
+       * See <a href="https://google-developers.appspot.com/chart/interactive/docs/reference#addrow">Google Visualization API reference (addRow)</a>
+       * for row format.
+       *
+       * @attribute rows
+       * @type array
+       */
+      rows: {
+        type: Array,
+        value: function() { return [] }
```
Comment (ebidel) [inline google-chart.html]: `return []; }`

Comment (ebidel) [inline google-chart.html]: Can you update throughout?

File: google-chart.html [hunk 14]
```diff
@@ -41,269 +41,357 @@
...
+    chartObject: null,
+
+    isReady: false,
+
+    canDraw: false,
+
+    dataTable: null,
+
+    created: function() {
+      this.chartTypes = {};
```
Comment (ebidel) [inline google-chart.html]: can this be a property with a default value too?

File: google-chart.html [hunk 15]
```diff
@@ -41,269 +41,357 @@
...
+      this.loadData();
+    },
+
+    selectionChanged: function() {
+      if (this.chartObject && this.setSelection) {
+        this.chartObject.setSelection(this.selection);
+      }
+    },
+
+    observers: [
```
Comment (ebidel) [inline google-chart.html]: can you list `observers` at the top?

**Comment by ebidel** [comment]
Reviewed

File: bower.json (hunk 1 above)
Comment (atotic) [inline bower.json]: fixed

File: bower.json (hunk 2 above)
Comment (atotic) [inline bower.json]: fixed

File: demo.html (hunk 3 above)
Comment (atotic) [inline demo.html]: fixed

File: demo.html (hunk 4 above)
Comment (atotic) [inline demo.html]: fixed. Making me work for old code :)

Comment (ebidel) [inline demo.html]: Yea sorry. I realize you didn't write it in the first place :) Now is the time!

File: demo.html (hunk 6 above)
Comment (atotic) [inline demo.html]: fixed.

File: demo.html (hunk 7 above)
Comment (atotic) [inline demo.html]: fixed.

File: demo.html (hunk 9 above)
Comment (atotic) [inline demo.html]: fixed.

File: demo.html (hunk 10 above)
Comment (atotic) [inline demo.html]: fixed.

File: google-chart.html (hunk 11 above)
Comment (atotic) [inline google-chart.html]: fixed

File: google-chart.html (hunk 12 above)
Comment (atotic) [inline google-chart.html]: fixed;

File: google-chart.html (hunk 13 above)
Comment (atotic) [inline google-chart.html]: fixed;;

File: google-chart.html (hunk 14 above)
Comment (atotic) [inline google-chart.html]: fixed.

File: google-chart.html (hunk 15 above)
Comment (atotic) [inline google-chart.html]: fixed.

**Comment by atotic** [comment]
Fixes checked in

**Comment by ebidel** [comment]
LGTM

**Pull request merged**